templates/repo_report_template.md

```diff
@@ -108,5 +108,10 @@ date_stampLastWeek: {date_stamp}
     <figure>
       <embed type="image/svg+xml" src="{{{{ "/assets/img/graphs/{repo_owner}/{repo_name}/commit_sparklines_{repo_name}_data.svg" | url }}}}" />
     </figure>
+    <div class="firstResponsePRCrop">
+      <figure>
+        <img alt="firstResponseForClosedPR" src="{{{{ "/assets/img/graphs/{repo_owner}/{repo_name}/firstResponseForClosedPR_{repo_name}_data.png" | url }}}}" />
```

Comment:
Will all repos have this graph? If a repo doesn't have any closed PRs or let alone PRs, will a graph still be generated for it?

If some repos lack this graph, I may have to add a function to check for file existence of the image because a broken image/placeholder image will be displayed

Reply:
That's a good question since we won't be able to download a graph if there isn't enough data to generate it. 

>If some repos lack this graph, I may have to add a function to check for file existence of the image because a broken image/placeholder image icon will be displayed if the file doesn't exist

That makes sense and that function is probably needed.

Reply:
Sounds good, will work on this. Should I add on to this PR or create a new PR? This is ready to ship otherwise